src/schema/settings.ts

```diff
@@ -118,6 +118,11 @@ export const typeDefs = /* GraphQL */ `
     """
     optOutWeeklyGoal: Boolean!
 
+    """
+    Whether the user opted out from reading streak
+    """
+    optOutReadingStreak: Boolean!
```

Comment:
Update the `GQLSettings` typescript definition as well

Reply:
updated